Author: Jasper Grimm
Add missing ControlsFX dependency
Use environment variable to point to JavaFX, ControlsFX install dirs
diff -Nru megan-ce.orig/antbuild/build.xml megan-ce/antbuild/build.xml
--- megan-ce.orig/antbuild/build.xml	2021-09-02 15:31:18.000000000 +0100
+++ megan-ce/antbuild/build.xml	2021-09-02 15:31:35.000000000 +0100
@@ -21,7 +21,9 @@
 		<property name="ant.build.javac.source" value="13"/>
 		<property name="ant.build.javac.target" value="13"/>
 
-		<property name="jfxDir" value="../../javafx/lib"/>
+                <property environment="env"/>
+                <property name="jfxDir" value="${env.JAVAFX_HOME}/lib"/>
+                <property name="cfxDir" value="${env.CONTROLSFX_HOME}/lib"/>
 		<property name="jlodaAntDir" value="../../jloda/antbuild"/>
 		<property name="buildSrcDir" value="../antbuild/src"/>
 		<property name="buildModulesDir" value="../antbuild/modules"/>
@@ -35,6 +37,7 @@
 			<fileset dir="../../jloda/jars"  includes="*.jar"/>
 			<fileset dir="../../megan-ce/jars"  includes="*.jar"/>
 			<fileset dir="${jfxDir}"  includes="*.jar"/>
+                        <fileset dir="${cfxDir}"  includes="*.jar"/>
 			<fileset dir="../antbuild"  includes="jloda.jar"/>
 		</path>
 
@@ -42,6 +45,7 @@
 			<fileset dir="../../jloda/jars"  includes="*.jar"/>
 			<fileset dir="../../megan-ce/jars"  includes="*.jar"/>
 			<fileset dir="${jfxDir}"  includes="*.jar"/>
+                        <fileset dir="${cfxDir}"  includes="*.jar"/>
 			<fileset dir="../antbuild"  includes="jloda.jar"/>
 			<fileset dir="../antbuild"  includes="megan.jar"/>
 		</path>
